### PR TITLE
Part 4. View single photo

### PR DESCRIPTION
**Pull request by korotkovoleksii**

### CODE REVIEW

File: src/core/App.tsx [hunk 1]
```diff
@@ -1,30 +1,27 @@
-import { Box, Divider, CssBaseline, Container, ThemeProvider } from '@mui/material';
+import { Box, CssBaseline, ThemeProvider } from '@mui/material';
 import { Route, Routes } from 'react-router-dom';
-import Header from 'modules/components/Header';
-import SideMenu from 'modules/components/SideMenu';
+// import Header from 'modules/components/Header';
```
Comment (leo-frontend) [inline src/core/App.tsx:3]: Do you need this?

File: src/modules/components/Header/Header.tsx [hunk 2]
```diff
@@ -1,36 +1,103 @@
...
 const Header = (): JSX.Element => {
-  return (
-    <Box
-      sx={{
-        pt: 2,
-        pb: 1,
-      }}
+  const { data: selectedPhoto, isShow: isSelectedInfoHederShow } = useAppSelector((state) => state.selectedPhotos);
+  const { pathname } = useLocation();
+  // eslint-disable-next-line no-console
```
Comment (leo-frontend) [inline src/modules/components/Header/Header.tsx:18]: Delete this

File: src/modules/components/ShowPhotosGrid/ShowPhotosGrid.tsx [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+import { useState } from 'react';
+import {
+  Box,
+  Checkbox,
+  Container,
+  Dialog,
+  IconButton,
+  ImageList,
+  ImageListItem,
+  // ImageListItemBar,
```
Comment (leo-frontend) [inline src/modules/components/ShowPhotosGrid/ShowPhotosGrid.tsx:11]: Remove

File: src/modules/pages/SelectPhotos.tsx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+  const body = (
+    <Box>
+      <ImageList cols={8} gap={8}>
+        {allPhotos.map((item) => (
+          <ImageListItem key={item.id} onClick={() => {
+            setSelectedPhotoId(selectedPhotoId.includes(item.id) ?
+              selectedPhotoId.filter(itemId => itemId !== item.id) : [...selectedPhotoId, item.id]);
+          }}
+          >
+            {/* <Checkbox checked={selectedPhotoId.includes(item.id)} sx={{ position: 'absolute', right: 0 }} /> */}
```
Comment (leo-frontend) [inline src/modules/pages/SelectPhotos.tsx:103]: Remove this

**Review by leo-frontend — CHANGES_REQUESTED** [review]

File: src/core/App.tsx (hunk 1 above)
Comment (korotkovoleksii) [reply src/core/App.tsx]: +

File: src/modules/components/Header/Header.tsx (hunk 2 above)
Comment (korotkovoleksii) [reply src/modules/components/Header/Header.tsx]: +

File: src/modules/components/ShowPhotosGrid/ShowPhotosGrid.tsx (hunk 3 above)
Comment (korotkovoleksii) [reply src/modules/components/ShowPhotosGrid/ShowPhotosGrid.tsx]: +

File: src/modules/pages/SelectPhotos.tsx (hunk 4 above)
Comment (korotkovoleksii) [reply src/modules/pages/SelectPhotos.tsx]: +

**Review by leo-frontend — APPROVED** [review]

**Pull request merged**